std/typecons.d

```diff
@@ -2522,6 +2527,12 @@ Returns:
         return _value;
     }
 
+    /// ditto
+    @property get(T)(T datum) inout @safe pure nothrow
```

Comment:
Can you change the `T` to `U` as it masks the overarching template type from the `Nullable` struct?